src/clojure/zero_one/geni/arrow.clj

```diff
@@ -0,0 +1,222 @@
+(ns zero-one.geni.arrow
+  (:import [org.apache.arrow.vector VarCharVector
```

Comment:
I believe you are getting unused imports from `clj-kondo`.

Reply:
fixed